NDTensors/src/lib/GradedAxes/src/fusion.jl

```diff
@@ -5,6 +5,7 @@ struct OneToOne{T} <: AbstractUnitRange{T} end
 OneToOne() = OneToOne{Bool}()
 Base.first(a::OneToOne) = one(eltype(a))
 Base.last(a::OneToOne) = one(eltype(a))
+BlockArrays.blockaxes(::OneToOne{Bool}) = (Block.(OneToOne()),)
```

Comment:
```suggestion
BlockArrays.blockaxes(::OneToOne) = (Block.(OneToOne()),)
```
unless there is a reason to only define it for `OneToOne` with element type `Bool` that I'm not seeing.

Reply:
Maybe it should be:
```julia
BlockArrays.blockaxes(a::OneToOne) = (Block.(OneToOne{eltype(a)}()),)
```
to make it more generic. I don't know if we need the output element type to match the input but I guess it doesn't hurt.

Reply:
`blockaxes` was only crashing in the `Bool` case, but indeed it can be made generic.

Reply:
I see, what issue was showing up in the `Bool` case? Could you make a comment about that so we can track it and maybe fix it in the future (say if it is an issue in `BlockArrays.jl`)?

Reply:
@ogauthe you resolved this issue but you didn't address my last suggestion about adding a comment on what issue this definition is fixing.

Reply:
Sorry I commited and resolved just after your first comment. `BlockArrays` defines `blockaxes` as `(Block.(Base.OneTo(one(T))),)`, but `Base.OneTo(true)` throws `ArgumentError`